STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,65 @@
+# Istio Steering Committee
+
+The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing. Steering was created to allow the Techincal Oversight Committee to exclusively focus on the technical aspects of the project. 
+
+## Steering Committee Charter
+
+1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio
+2. Charter and refine policy for community groups (including Working Groups and Committees)
+3. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
+4. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository
+5. Control and delegate access to and establish processes regarding other project resources/assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
+6. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
+7. Resolve any dispute from the Technical Oversight Committee.
+
+## Steering Committee Members
+
+Seats in Steering are held by a company, not by the individual users. Currently Steering consists of members from Google & IBM. These are:
+
+* [April Nassi](https://github.com/thisisnotapril) (Google)
+* [Dan Berg](https://github.com/dcberg) (IBM)
+* [Dan Ciruli](https://github.com/oaktowner) (Google)
+* Jennifer Lin (Google)
+* [Lin Sun](https://github.com/linsun) (IBM)
+* [Louis Ryan](https://github.com/louiscryan) (Google)
+* [Martin Taillefer](https://github.com/geeknoid) (Google)
+* Nilesh Patel (IBM)
+* [Sven Mawson](https://github.com/smawson) (Google)
+
+## General Questions
+
+*Why Are Seats Company Held?*
+
+Steering was created as the Istio project was in infancy, in order to tackle governance and overall project strategy. To date Istio 1.0 has the bulk of contributions coming from the project founders, Google & IBM. Together it was decided that having strong corporate leadership was needed for the project to ensure continued velocity. Steering was created with 9 seats held, and with 11 being planned.
+
+*Will Steering Always Be Just IBM & Google?*
+
+That is not the intent. The plan is to add additional seats to Steering. We have not yet decided how that will be done. Your feedback is appreciated. We do want to ensure that those contributing to the project are properly represented. At a minimum, the membership is reviewed once a year.
```

Comment:
This question might be better generalized to "How are Steering seats allocated?" the answer to which I'd paraphrase in line 37 is "based upon representation by companies based upon their contributions, although no final decision has been made on how this will be done."

The good thing about this is this would permit removal of the negative "This is not the intent" language.

Cheers
-steve